utils/ap51-flash/Makefile

```diff
@@ -7,15 +7,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ap51-flash
-PKG_VERSION:=2018.0
+PKG_VERSION:=2019.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://github.com/ap51-flash/ap51-flash/releases/download/v$(PKG_VERSION)
-PKG_HASH:=e38e48a12d7c7b8e189f5538b78bbf00548044414d9ededa18ec9a5b5886afaa
+PKG_HASH:=e7992b2151721cc6f5db91f443ad7fc83cb5604c08cd11fca3e78ecd6b538e57
 PKG_MAINTAINER:=Russell Senior <russell@personaltelco.net>
-PKG_LICENSE:=GPL-3.0+
-PKG_LICENSE_FILES:=LICENSES/preferred/GPL-3.0
+PKG_LICENSE:=GPL-3.0+ CC0-1.0
```

Comment:
should be GPL-3.0-or-later. See: https://spdx.org/licenses/

Reply:
First thing: This was not added in this commit

Other than that, GPL-3.0-or-later is the SPDX version but OpenWrt doesn't seem to use the SPDX version in the main repository. And there is a mixture of both in the packages repository (beside having different requirements in CONTRIBUTING.md). 

So it might be a good idea to have a treewide change for using the GPL-3.0-or-later form